.github/workflows/ci-build.yml

```diff
@@ -34,3 +34,22 @@ jobs:
         python-version: '3.6'
     - name: Run Unit Tests
       run: bash qa/runtest.sh
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v1
+      with:
+        token: ${{ secrets.CODECOV_TOKEN }}
+        file: ./coverage.xml
+  openshift-yaml-validate:
```

Comment:
Thats correct, but it does run in parallel and doesn't take lot of time to complete.

Reply:
Also making it optional may be an issue during true positive case. Not so much worried to optimise this step because it hardly takes 10s and which runs parallel to other checks.